my-calendar/src/java/org/sakaiproject/widgets/mycalendar/ui/WidgetPage.java

```diff
@@ -76,9 +76,9 @@ public void renderHead(final IHeaderResponse response) {
 		// for datepicker
 		response.render(new PriorityHeaderItem(
 				JavaScriptHeaderItem.forReference(getApplication().getJavaScriptLibrarySettings().getJQueryReference())));
-		response.render(CssHeaderItem.forUrl(String.format("/my-calendar/styles/jquery-ui.min.css?version=%s", version)));
-		response.render(JavaScriptHeaderItem.forUrl(String.format("/my-calendar/scripts/jquery-ui.min.js?version=%s", version)));
-		response.render(JavaScriptHeaderItem.forUrl(String.format("/my-calendar/scripts/moment.js?version=%s", version)));
+		response.render(JavaScriptHeaderItem.forUrl(String.format("/library/webjars/jquery-ui/1.11.3/jquery-ui.min.css?version=%s", version)));
```

Comment:
Good catch.  I am surprised it worked with it my way.

-- 
Mitch Golden
CTO, Noodle Partners

On July 3, 2016 2:36:32 AM EDT, Steve Swinsburg <notifications@github.com> wrote:
>> @@ -76,9 +76,9 @@ public void renderHead(final IHeaderResponse
>response) {
>>  		// for datepicker
>>  		response.render(new PriorityHeaderItem(
>> 
>				JavaScriptHeaderItem.forReference(getApplication().getJavaScriptLibrarySettings().getJQueryReference())));
>>
>-		response.render(CssHeaderItem.forUrl(String.format("/my-calendar/styles/jquery-ui.min.css?version=%s",
>version)));
>>
>-		response.render(JavaScriptHeaderItem.forUrl(String.format("/my-calendar/scripts/jquery-ui.min.js?version=%s",
>version)));
>>
>-		response.render(JavaScriptHeaderItem.forUrl(String.format("/my-calendar/scripts/moment.js?version=%s",
>version)));
>>
>+		response.render(JavaScriptHeaderItem.forUrl(String.format("/library/webjars/jquery-ui/1.11.3/jquery-ui.min.css?version=%s",
>version)));
>
>This should still be `CssHeaderItem` - will fix afterwards.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/steveswinsburg/sakai-widgets/pull/26/files/308610c2e1bd4f2647e6fbb978542de55c52f040#r69386300